src/utils/ResultListUtils.ts

```diff
@@ -23,6 +23,23 @@ export class ResultListUtils {
     return resultList ? !!resultList.options.enableInfiniteScroll : false;
   }
 
+  public static scrollToTop(root: HTMLElement) {
+    const resultList = ResultListUtils.getActiveResultList(root);
+    if (!resultList) {
+      new Logger(this).warn('No active ResultList, scrolling to the top of the Window');
+      return window.scrollTo(0, 0);
+    }
+
+    const scrollContainer = resultList.options.infiniteScrollContainer;
+
+    if (typeof scrollContainer.scrollTo === 'function') {
+      const searchInterfacePosition = resultList.searchInterface.element.getBoundingClientRect().top;
+      scrollContainer.scrollTo(0, window.pageYOffset + searchInterfacePosition);
+    } else {
+      (<HTMLElement>scrollContainer).scrollTop = 0;
```

Comment:
Rather than casting, I prefer to check that if `scrollContainer` is an  `HTMLElement`. I see there are browser incompatibilities with doing a simple `instanceof HTMLElement` but there is this code which we could place in one of the util files:

```
//Returns true if it is a DOM element    
function isElement(o){
  return (
    typeof HTMLElement === "object" ? o instanceof HTMLElement : //DOM2
    o && typeof o === "object" && o !== null && o.nodeType === 1 && typeof o.nodeName==="string"
);
}
```

https://stackoverflow.com/questions/384286/javascript-isdom-how-do-you-check-if-a-javascript-object-is-a-dom-object

Reply:
This piece of code was there for a while in the ResultList component, I just moved it. We already know the scrollContainer is a htmlElement the cast is to prevent

It will basically call the scrollTo method if it exists (it will exist on every html element and the window).
In the case of the IE11 browser, it won't exist so we default to the scrollTop property.

I'm not a 100% sure that snippet is necessary here, but it could be useful in other use cases.